runtime/autoload/provider/clipboard.vim

```diff
@@ -156,6 +156,13 @@ function! provider#clipboard#Executable() abort
     let s:copy['*'] = s:copy['+']
     let s:paste['*'] = s:paste['+']
     return 'tmux'
+  elseif executable('pbcopy') && executable('pbpaste') 
```

Comment:
tmux should always be the last case. So this check should go before the tmux case.

Reply:
OK, I have moved it before the tmux case.